interfaces/prompting/prompting.go

```diff
@@ -145,7 +140,7 @@ func (lifespan *LifespanType) UnmarshalJSON(data []byte) error {
 	case LifespanForever, LifespanSingle, LifespanTimespan:
 		*lifespan = value
 	default:
-		return fmt.Errorf(`cannot have lifespan other than %q, %q, or %q: %q`, LifespanForever, LifespanSingle, LifespanTimespan, value)
+		return fmt.Errorf(`%w: cannot have lifespan other than %q, %q, or %q: %q`, ErrInvalidLifespan, LifespanForever, LifespanSingle, LifespanTimespan, value)
```

Comment:
because of that point I'm not sure if simply wrapping this way is the right approach